adafruit_minimqtt/adafruit_minimqtt.py

```diff
@@ -51,7 +51,7 @@
 
 from .matcher import MQTTMatcher
 
-__version__ = "0.0.0+auto.0"
+__version__ = "7.6.3"
```

Comment:
Do not change this. The version is filled in when the release artifacts are built.

```suggestion
__version__ = "0.0.0+auto.0"
```